### ISSUE TITLE
babel-helper-vue-jsx-merge-props missing in dependencies?

### ISSUE THREAD

**Issue by stomaskov** [cristijora/vue-tabs#44]
Did `npm install vue-nav-tabs` and it was missing.

`vue-tabs.common.js:12 Uncaught Error: Cannot find module "babel-helper-vue-jsx-merge-props"
    at webpackMissingModule (vue-tabs.common.js:12)
    at eval (vue-tabs.common.js:12)
    at Object.../node_modules/vue-nav-tabs/dist/vue-tabs.common.js (index.bundle.js:82)
    at __webpack_require__ (index.bundle.js:20)
    at eval (header.js:3)
    at Object../header.js (index.bundle.js:116)
    at __webpack_require__ (index.bundle.js:20)
    at eval (index.js:9)
    at Object../index.js (index.bundle.js:128)
    at __webpack_require__ (index.bundle.js:20)
webpackMissingModule @ vue-tabs.common.js:12
(anonymous) @ vue-tabs.common.js:12
../node_modules/vue-nav-tabs/dist/vue-tabs.common.js @ index.bundle.js:82
__webpack_require__ @ index.bundle.js:20
(anonymous) @ header.js:3
./header.js @ index.bundle.js:116
__webpack_require__ @ index.bundle.js:20
(anonymous) @ index.js:9
./index.js @ index.bundle.js:128
__webpack_require__ @ index.bundle.js:20
(anonymous) @ index.bundle.js:69
(anonymous) @ index.bundle.js:72
`

**Comment by cristijora** [comment]
Hi @stomaskov 
just make sure you `import Tabs from 'vue-nav-tabs/dist/vue-tabs.js` which is the ES5 compiled version that should give you any warnings like this. 
If you `import Tabs from 'vue-nav-tabs'` it will most likely get the raw version which is written with jsx

**Comment by stomaskov** [comment]
Yep, importing from dist works.
Is this environment related or something else?
Because documentation doesn't mention importing from dist.

**Comment by cristijora** [comment]
Yeah this can be environment related. I will check if the `main` file in package.json is set correctly as it might cause this issue.

**Comment by petemolinero** [comment]
Perhaps the [documentation](https://cristijora.github.io/vue-tabs/#/?id=component-registration) should suggest `import Tabs from 'vue-nav-tabs/dist/vue-tabs.js` instead of `import VueTabs from 'vue-nav-tabs'`?

**Comment by genert** [comment]
Pump! 👍
